gtk4/src/subclass/combo_box.rs

```diff
@@ -0,0 +1,78 @@
+use glib::subclass::prelude::*;
+use glib::translate::*;
+use glib::{Cast, GString};
+
+use super::widget::WidgetImpl;
+use crate::{ComboBox, Widget};
+
+pub trait ComboBoxImpl: ComboBoxImplExt + WidgetImpl {
+    fn changed(&self, combo_box: &Self::Type) {
+        self.parent_changed(combo_box)
+    }
+    fn format_entry_text(&self, combo_box: &Self::Type, path: &str) -> Option<GString> {
+        self.parent_format_entry_text(combo_box, path)
+    }
+}
+
+pub trait ComboBoxImplExt: ObjectSubclass {
+    fn parent_changed(&self, combo_box: &Self::Type);
+
+    fn parent_format_entry_text(&self, combo_box: &Self::Type, path: &str) -> Option<GString>;
```

Comment:
Did you check if `path` is indeed not `nullable` and the return value is?

Reply:
The return value can be null if the ComboBox doesn't set a `entry-text-column` property. Although I didn't find any examples of usages with NULL passed as `path`.

Reply:
Ok, worst case it will panic and can be fixed then :)